domains/_vercel.risuu.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "wrisuu",
+        "discord_username": "wrisuu",
+        "discord_id": "730160704409305088",
```

```suggestion
        "discord": "730160704409305088",
```
its better to use the Discord ID instead of username as it can be change.